### PR TITLE
Logger

### PR DESCRIPTION
**Pull request by VitalikVoicix**
Ability to log our environment when we start our application with npm start

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,27 +1,28 @@
-MONGO_URL=mongodb://root:root@localhost:27017/horondi?authSource=admin
-MAIL_USER=horondi@outlook.com
-MAIL_PASS=487Horondi512
-SECRET=4hG7agm$1z5hkiyb4msu5ka84gs5ruy8kfn3u7$8kbd4e5d2j7n6v3jp88gsn58mhd84
-MAIL_HOST=smtp-mail.outlook.com
-MAIL_PORT=587
+MONGO_URL=mongodb+srv://user-read-write1:sfd$42A3a@horondi-yl27g.mongodb.net/horondi-23?retryWrites=true&w=majority
```
Comment (TorskyM) [inline .env:1]: Remove this file, also `MONGO_URL` in `test` should refer to the locale db

Comment (VitalikVoicix) [reply .env]: Allready remove.

**Review by DmytroDidukh — APPROVED** [review]

File: package-lock.json [hunk 2]
```diff
@@ -4175,12 +4175,15 @@
             "ws": "^6.0.0"
           },
           "dependencies": {
+<<<<<<< HEAD
```
Comment (ahd3r) [inline package-lock.json:4178]: Why you did not resolve conflicts?

File: logger.js [hunk 3]
```diff
@@ -5,13 +5,12 @@ const logger = winston.createLogger({
   level: 'info',
   format: winston.format.combine(
     winston.format.timestamp(),
-    winston.format.json()
```
Comment (ahd3r) [inline logger.js:8]: You do not have to remove the previous logging, you had to create a new one, to logging an `env`

File: server.js [hunk 4]
```diff
@@ -32,14 +33,6 @@ const server = new ApolloServer({
   context: async ({ req }) => {
     const { token } = req.headers || '';
 
-    logger.log({
```
Comment (ahd3r) [inline server.js:35]: this is a logger for income request, it has to be here

File: package-lock.json (hunk 2 above)
Comment (VitalikVoicix) [reply package-lock.json:4178]: Allready resolved

File: logger.js (hunk 3 above)
Comment (VitalikVoicix) [reply logger.js:7]: Add back old logger

File: server.js (hunk 4 above)
Comment (VitalikVoicix) [reply server.js:29]: Put him on this place

File: loggerInFile.js [hunk 5]
```diff
@@ -1,16 +1,16 @@
 const winston = require('winston');
```
Comment (ahd3r) [inline loggerInFile.js:1]: it is better to rename this file back to `logger.js` and create `loggerEnv` here.

File: loggerInFile.js [hunk 6]
```diff
@@ -1,16 +1,16 @@
...
-
-const logger = winston.createLogger({
+const loggerInFile = winston.createLogger({
```
Comment (ahd3r) [inline loggerInFile.js:2]: Rename this variable to `loggerHttp`

Comment (VitalikVoicix) [reply loggerInFile.js]: Allready renamed

File: loggerInFile.js (hunk 5 above)
Comment (VitalikVoicix) [reply loggerInFile.js]: rename back to logger,js.Dont know why,but  I cant create two loggers in one file becaouse they have some conflicts between each other.

**Review by TorskyM — APPROVED** [review]

**Review by ahd3r — APPROVED** [review]

**Pull request merged**